src/check/structure.test.js

```diff
@@ -1,11 +1,14 @@
 import {isDir, isFile} from './lib';
 import tap from 'tap';
 
-tap.ok(isDir('scripts'), 'the "scripts/" directory exists');
 tap.ok(isDir('src'), 'the "src/" directory exists');
-tap.ok(isFile('src/plugin.js'), 'the "src/plugin.js" file exists');
+tap.ok(isDir('src/js/'), 'the "src/js/" directory exists');
+tap.ok(isFile('src/js/index.js'), 'the "src/plugin.js" file exists');
```

Comment:
seems like there should be an array of `files` and `dirs` to check, that way we won't have an issue with diverging descriptions.